docs/tutorials/kubernetes-basics/index.html

```diff
@@ -15,66 +15,74 @@
+        <h2>Kubernetes Basics</h2>
+        <p>This tutorial provides a walkthrough of the basics of the Kubernetes cluster orchestration system. Each module contains some background information on major Kubernetes features and concepts, and includes an interactive online tutorial. These interactive tutorials let you manage a simple cluster and its containerized applications for yourself.</p>
+        <p>Using the interactive tutorials, you can learn to:</p>
+        <ul>
+        <li>Deploy a containerized application on a cluster</li>
+        <li>Scale the deployment</li>
+        <li>Update the containerized application with a new software version</li>
+        <li>Debug the containerized application</li>
+        </ul>
+        <p>The tutorials use Katacoda to run a virtual terminal in your web browser that runs Minikube, a small-scale local deployment of Kubernetes that can run anywhere. There's no need to install any software or configure anything; each interactive tutorial runs directly out of your web browser itself.</p>
       </div>
     </div>
 
     <br>
 
     <div class="row">
       <div class="col-md-9">
-        <h2>Why Kubernetes?</h2>
-        <p>Today users expect applications to be available 24/7, while developers expect to deploy new versions of those applications several times a day. The way we build software is moving in this direction, enabling applications to be released and updated in an easy and fast way without downtime. We also need to be able to scale application in line with the user demand and we expect them to make intelligent use of the available resources. <a href="http://kubernetes.io/docs/whatisk8s/">Kubernetes</a>  is a platform designed to meet those requirements, using the experience accumulated by Google in this area, combined with best-of-breed ideas from the community.</p>
+        <h2>What can Kubernetes do for you?</h2>
+        <p>With modern web services, users expect applications to be available 24/7, and developers expect to deploy new versions of those applications several times a day. Containzerization helps package software to serve these goals, enabling applications to be released and updated in an easy and fast way without downtime. Kubernetes helps you make sure those containerized applications run where and when you want, and helps them find the resources and tools they need to work. <a href="http://kubernetes.io/docs/whatisk8s/">Kubernetes</a>  is a production-ready, open source platform designed with the Google's accumulated experience in container orchestration, combined with best-of-breed ideas from the community.</p>
       </div>
     </div>
 
     <div class="content__modules">
-      <h2>Getting Started Modules</h2>
+      <h2>Kubernetes Basics Modules</h2>
```

Comment:
I suppose we could call it "Modules in Kubernetes Basics", but that's a little stilted.